moved the menu to this part  #OICI-180

Iandraina-etech opened the pull request:
moved Prescriber ans Specialities to this  menu instead of claim

Iandraina-etech committed:
Moved this menu from Claim to Admin

Added the prescriber and specialities to this menu

Iandraina-etech committed:
Merge pull request #1 from Iandraina-etech/feature-arkeup

Moved this menu from Claim to Admin

Iandraina-etech committed:
Update AdminMainMenu.js

Removed Comment

mngoe commented on src/components/AdminMainMenu.js:
```diff
@@ -40,6 +44,21 @@ class AdminMainMenu extends Component {
     const { rights } = this.props;
     const entries = [];
 
+    if (!!rights.filter((r) => r >= RIGHT_CLAIMREVIEW && r <= RIGHT_PROCESS).length) {
```
Comme spécifié dans la demande initiale : 
En tant qu’administrateur openIMIS, je veux pouvoir lister, filtrer, rechercher, ajouter, modifier et supprimer des prescripteurs, afin de maintenir une base de données à jour.

Je n'ai pas vérifié fonctionnellement , mais apparement vous autorisez ce menu pour les Claim Review.

Iandraina-etech committed:
Updated the rights to match the User creatyion right

Adapted the rights verifications to match the Administrator Rights
Used the User creation rights

Iandraina-etech committed:
Merge branch 'feature-arkeup' into develop-comores

Iandraina-etech committed:
Merge pull request #2 from Iandraina-etech/develop-comores

Develop comores

Iandraina-etech committed:
Merge pull request #3 from Iandraina-etech/feature-arkeup

Merge pull request #2 from Iandraina-etech/develop-comores

Iandraina-etech committed:
Update constants.js

Iandraina-etech committed:
Merge pull request #4 from Iandraina-etech/feature-arkeup

Update constants.js
